.mage/docs.go

```diff
@@ -23,18 +24,41 @@ import (
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/magefile/mage/target"
 )
 
 // Docs namespace
 type Docs mg.Namespace
 
 func execHugo(args ...string) error {
-	return execGo("run", append([]string{"github.com/gohugoio/hugo", "-s", "./doc"}, args...)...)
+	return execGo("run", append([]string{"-tags", "extended", "github.com/gohugoio/hugo", "-s", "./doc"}, args...)...)
 }
 
-// Deps installs documentation dependencies.
-func (Docs) Deps() error {
-	return sh.RunV("git", "submodule", "update", "--init", "doc/themes/hugo-theme-techdoc")
+func (d Docs) yarn() (func(args ...string) error, error) {
```

Comment:
We probably already have this in JS namespace, right? Why not just reuse that one?

Reply:
We actually have 2, one for JS and one for JS SDK because we're executing yarn from a different workdir.

Reply:
Can't we just have a `func yarn(wd string) (func(args ...string) error, error)`?